src/components/QuizContainerNew/QuestionsListNew/QuestionListItemNew/QuizImage.tsx

```diff
@@ -21,41 +29,63 @@ const QuizImage: FC<ImageProps> = ({ src, alt }) => {
   };
 
   const [dims, setDims] = useState(calcDims(src.width, src.height));
-
+  const containerBackgroundColor = answerIsCorrect ? "lemon50" : "white";
```

Comment:
I think you've implemented this in the wrong component and thereby effected other functionality. Can we have a chat about this.